<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Thomas Kais</author>
 <comment>
 #------------------------------------------------------------
 # Copyright (c) 2018 Thomas Kais
 # Copyright (c) 2023 NicsMocs
 #
 # This file is subject to the terms and conditions defined in
 # file 'COPYING', which is part of this source code package.
 #------------------------------------------------------------
 </comment>
 <class>PreferencesDialog</class>
 <widget class="QDialog" name="PreferencesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>851</width>
    <height>638</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayoutDialog">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>833</width>
        <height>584</height>
       </rect>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <layout class="QVBoxLayout" name="verticalLayoutScrollArea">
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <property name="currentIndex">
          <number>2</number>
         </property>
         <widget class="QWidget" name="tabGeneral">
          <property name="mouseTracking">
           <bool>true</bool>
          </property>
          <attribute name="title">
           <string>General</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayoutTabGeneral">
           <item>
            <layout class="QGridLayout" name="gridLayoutTabGeneral" columnstretch="0,1,2,2,1,0">
             <item row="9" column="1">
              <widget class="QCheckBox" name="chbGrayscale">
               <property name="text">
                <string>monochrome</string>
               </property>
              </widget>
             </item>
             <item row="8" column="1">
              <widget class="QComboBox" name="cmbCameraMode"/>
             </item>
             <item row="4" column="1">
              <widget class="QCheckBox" name="chbButtons">
               <property name="text">
                <string>(enable/disable)</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="lblGeneral">
               <property name="font">
                <font>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>FotoBox</string>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="lblButtons">
               <property name="text">
                <string>buttons</string>
               </property>
               <property name="buddy">
                <cstring>chbButtons</cstring>
               </property>
              </widget>
             </item>
             <item row="8" column="0">
              <widget class="QLabel" name="lblCameraMode">
               <property name="text">
                <string>camera mode</string>
               </property>
               <property name="buddy">
                <cstring>cmbCameraMode</cstring>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="lblCountdown">
               <property name="text">
                <string>countdown</string>
               </property>
               <property name="buddy">
                <cstring>spbCountdown</cstring>
               </property>
              </widget>
             </item>
             <item row="8" column="2">
              <widget class="QToolButton" name="btnCameraModeReload"/>
             </item>
             <item row="1" column="1" colspan="3">
              <widget class="QLineEdit" name="txtPhotoFolder">
               <property name="focusPolicy">
                <enum>Qt::NoFocus</enum>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="lblPhotoName">
               <property name="text">
                <string>photo name</string>
               </property>
               <property name="buddy">
                <cstring>txtPhotoName</cstring>
               </property>
              </widget>
             </item>
             <item row="5" column="0">
              <widget class="QLabel" name="lblBackground">
               <property name="text">
                <string>background color</string>
               </property>
               <property name="buddy">
                <cstring>btnChooseColorBG</cstring>
               </property>
              </widget>
             </item>
             <item row="2" column="1" colspan="3">
              <widget class="QLineEdit" name="txtPhotoName"/>
             </item>
             <item row="6" column="0">
              <widget class="QLabel" name="lblCamera">
               <property name="font">
                <font>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Camera</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QSpinBox" name="spbCountdown">
               <property name="suffix">
                <string> seconds</string>
               </property>
               <property name="maximum">
                <number>9</number>
               </property>
              </widget>
             </item>
             <item row="3" column="2" colspan="2">
              <widget class="QLineEdit" name="txtShowColorCD">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="focusPolicy">
                <enum>Qt::NoFocus</enum>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="9" column="0">
              <widget class="QLabel" name="lblGrayscale">
               <property name="text">
                <string>show photo</string>
               </property>
               <property name="buddy">
                <cstring>chbGrayscale</cstring>
               </property>
              </widget>
             </item>
             <item row="1" column="5">
              <widget class="QToolButton" name="btnClearDirectoryContent"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="lblPhotoFolder">
               <property name="text">
                <string>photo folder</string>
               </property>
               <property name="buddy">
                <cstring>btnChooseDirectory</cstring>
               </property>
              </widget>
             </item>
             <item row="5" column="1" colspan="3">
              <widget class="QLineEdit" name="txtShowColorBG">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="focusPolicy">
                <enum>Qt::NoFocus</enum>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="7" column="0" colspan="6">
              <widget class="QLabel" name="lblCameraInfo">
               <property name="text">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;If you have a &lt;span style=&quot; font-style:italic;&quot;&gt;DSLR camera&lt;/span&gt;, choose &amp;quot;&lt;span style=&quot; font-weight:600;&quot;&gt;gphoto2&lt;/span&gt;&amp;quot; mode. If you want to use the &lt;span style=&quot; font-style:italic;&quot;&gt;Raspberry Pi Camera Module&lt;/span&gt;, choose &amp;quot;&lt;span style=&quot; font-weight:600;&quot;&gt;raspistill&lt;/span&gt;&amp;quot; mode.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <property name="wordWrap">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="8" column="3" rowspan="2" colspan="3">
              <widget class="QLabel" name="lblCameraModeInfo">
               <property name="alignment">
                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
               </property>
               <property name="wordWrap">
                <bool>true</bool>
               </property>
               <property name="openExternalLinks">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="4">
              <widget class="QToolButton" name="btnChooseDirectory"/>
             </item>
             <item row="3" column="4" colspan="2">
              <widget class="QToolButton" name="btnChooseColorCD">
               <property name="text">
                <string>Chose Color</string>
               </property>
               <property name="toolButtonStyle">
                <enum>Qt::ToolButtonTextOnly</enum>
               </property>
              </widget>
             </item>
             <item row="5" column="4" colspan="2">
              <widget class="QToolButton" name="btnChooseColorBG">
               <property name="text">
                <string>Chose Color</string>
               </property>
               <property name="toolButtonStyle">
                <enum>Qt::ToolButtonTextOnly</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacerTabGeneral">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tabPrint">
          <attribute name="title">
           <string>Print</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayoutTabPrint">
           <item>
            <layout class="QGridLayout" name="gridLayoutTabPrint">
             <item row="2" column="0">
              <widget class="QLabel" name="lblPrinterName">
               <property name="text">
                <string>choose printer</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QComboBox" name="cmbPrinterName"/>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="lblPrintSetup">
               <property name="font">
                <font>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Print Setup</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="lblPrint">
               <property name="text">
                <string>print photos</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QCheckBox" name="chbPrint">
               <property name="text">
                <string>(enable/disable)</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacerTabPrint">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tabExpert">
          <property name="mouseTracking">
           <bool>true</bool>
          </property>
          <attribute name="title">
           <string>Expert</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayoutTabExpert">
           <item>
            <layout class="QGridLayout" name="gridLayoutTabExpert" columnstretch="0,0,0,0,0">
             <item row="2" column="0">
              <widget class="QLabel" name="lblOutputPin">
               <property name="text">
                <string>output PIN</string>
               </property>
               <property name="buddy">
                <cstring>spbOutputPin</cstring>
               </property>
              </widget>
             </item>
             <item row="1" column="3" rowspan="3" colspan="2">
              <widget class="QLabel" name="lblPinInfo">
               <property name="text">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;a href=&quot;http://abyz.me.uk/rpi/pigpio/index.html#Type_1&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;Type 1&lt;/span&gt;&lt;/a&gt; - Model B (original model)&lt;br/&gt;&lt;a href=&quot;http://abyz.me.uk/rpi/pigpio/index.html#Type_2&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;Type 2&lt;/span&gt;&lt;/a&gt; - Model A, B (revision 2)&lt;br/&gt;&lt;a href=&quot;http://abyz.me.uk/rpi/pigpio/index.html#Type_3&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;Type 3&lt;/span&gt;&lt;/a&gt; - Model A+, B+, Pi Zero, Pi2B, Pi3B&lt;br/&gt;&lt;a href=&quot;http://abyz.me.uk/rpi/pigpio/index.html#CM&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;Compute Module&lt;/span&gt;&lt;/a&gt; - Compute Module&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
               </property>
               <property name="wordWrap">
                <bool>true</bool>
               </property>
               <property name="openExternalLinks">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="6" column="1">
              <widget class="QSpinBox" name="spbTimout">
               <property name="suffix">
                <string> seconds</string>
               </property>
              </widget>
             </item>
             <item row="5" column="0">
              <widget class="QLabel" name="lblArgumentLine">
               <property name="text">
                <string>argument line</string>
               </property>
               <property name="buddy">
                <cstring>txtArgumentLine</cstring>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QSpinBox" name="spbQueryInterval">
               <property name="accelerated">
                <bool>true</bool>
               </property>
               <property name="suffix">
                <string> milliseconds</string>
               </property>
               <property name="maximum">
                <number>1000</number>
               </property>
              </widget>
             </item>
             <item row="9" column="1">
              <widget class="QLineEdit" name="lineEdit_cloud_password">
               <property name="inputMethodHints">
                <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
               </property>
               <property name="echoMode">
                <enum>QLineEdit::Password</enum>
               </property>
              </widget>
             </item>
             <item row="0" column="2" colspan="3">
              <widget class="QLabel" name="lblPigpioDeamon">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QSpinBox" name="spbOutputPin">
               <property name="maximum">
                <number>40</number>
               </property>
              </widget>
             </item>
             <item row="9" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Passwort</string>
               </property>
              </widget>
             </item>
             <item row="7" column="0">
              <widget class="QLabel" name="label">
               <property name="font">
                <font>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Cloud</string>
               </property>
              </widget>
             </item>
             <item row="7" column="1">
              <widget class="QCheckBox" name="checkBox_cloud">
               <property name="text">
                <string>enable/disable upload Cloud</string>
               </property>
              </widget>
             </item>
             <item row="8" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Username</string>
               </property>
              </widget>
             </item>
             <item row="8" column="1">
              <widget class="QLineEdit" name="lineEdit_cloud_username"/>
             </item>
             <item row="1" column="1">
              <widget class="QSpinBox" name="spbInputPin">
               <property name="maximum">
                <number>40</number>
               </property>
              </widget>
             </item>
             <item row="10" column="0">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>URL</string>
               </property>
              </widget>
             </item>
             <item row="6" column="0">
              <widget class="QLabel" name="lblTimeout">
               <property name="text">
                <string>timeout value</string>
               </property>
               <property name="buddy">
                <cstring>spbTimout</cstring>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="lblQueryInterval">
               <property name="text">
                <string>query interval</string>
               </property>
               <property name="buddy">
                <cstring>spbQueryInterval</cstring>
               </property>
              </widget>
             </item>
             <item row="5" column="4">
              <widget class="QToolButton" name="btnArgumentLineHelp"/>
             </item>
             <item row="10" column="1">
              <widget class="QLineEdit" name="lineEdit_cloud_url"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="lblInputPin">
               <property name="text">
                <string>input PIN</string>
               </property>
               <property name="buddy">
                <cstring>spbInputPin</cstring>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="lblCameraExpert">
               <property name="font">
                <font>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Camera</string>
               </property>
              </widget>
             </item>
             <item row="11" column="0">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Compress Pics</string>
               </property>
              </widget>
             </item>
             <item row="5" column="1" colspan="2">
              <widget class="QLineEdit" name="txtArgumentLine">
               <property name="clearButtonEnabled">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="lblBuzzer">
               <property name="font">
                <font>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Buzzer</string>
               </property>
              </widget>
             </item>
             <item row="12" column="0">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Size</string>
               </property>
              </widget>
             </item>
             <item row="11" column="1">
              <widget class="QCheckBox" name="checkBox_cloud_compress">
               <property name="text">
                <string>enable/disable before uploading</string>
               </property>
              </widget>
             </item>
             <item row="12" column="1">
              <widget class="QSpinBox" name="spinBox_cloud_compress_size">
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>2000</number>
               </property>
               <property name="value">
                <number>500</number>
               </property>
              </widget>
             </item>
             <item row="12" column="2">
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>kByte</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacerTabExpert">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacerScrollArea">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close|QDialogButtonBox::Help|QDialogButtonBox::Ok|QDialogButtonBox::RestoreDefaults</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>btnChooseDirectory</tabstop>
  <tabstop>btnClearDirectoryContent</tabstop>
  <tabstop>txtPhotoName</tabstop>
  <tabstop>spbCountdown</tabstop>
  <tabstop>btnChooseColorCD</tabstop>
  <tabstop>chbButtons</tabstop>
  <tabstop>btnChooseColorBG</tabstop>
  <tabstop>cmbCameraMode</tabstop>
  <tabstop>btnCameraModeReload</tabstop>
  <tabstop>chbGrayscale</tabstop>
  <tabstop>chbPrint</tabstop>
  <tabstop>cmbPrinterName</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
